fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/aggregate/StreamGrouping.java

```diff
@@ -193,8 +193,4 @@ private Object evalGroupingKey(@Nullable final Object currentObject) {
         final EvaluationContext nestedContext = context.withBinding(Bindings.Internal.CORRELATION, alias, currentObject);
         return Objects.requireNonNull(groupingKeyValue).eval(store, nestedContext);
     }
-
-    public boolean isResultOnEmpty() {
```

Comment:
If we go with this as is, I believe it means that deserialized plans that were created on older versions will have different semantics than they were originally planned. So, for example, a streaming aggregate with a null grouping that was planned without a `DefaultOnEmpty` plan on top will go from producing a single result to producing no values at all. That only becomes relevant if there's a continuation, which may not actually be possible here, but it feels a bit iffy.

If we choose not to make the cross-version compatibility work, we should at least call out how it's been broken.